### ISSUE TITLE
Level: robocopy vs eacopy

### ISSUE THREAD

**Issue by sergeevabc** [electronicarts/EACopy#10]
``` console
C:\SOURCE
└── MEH

robocopy C:\SOURCE D:\BACKUP\SOURCE /e /xf * /lev:1

D:\BACKUP
└── SOURCE
```
but
``` console
eacopy C:\SOURCE D:\BACKUP\SOURCE /e /xf * /lev:1

D:\BACKUP       
└── SOURCE   
    └── MEH
```
Why `eacopy` does not respect `lev` switch here?

**Comment by ben-may** [comment]
@honkstar1 Any chance you are still working with/on EACopy and want to look into this issue?

**Comment by sergeevabc** [comment]
@ben-may, not sure what you mean. I reported steps to reproduce the issue, asked the question about it, and has been waiting for an answer or a fixed binary to try.

**Comment by ben-may** [comment]
@sergeevabc I meant that @honkstar1 is the expert here, I wanted to know if he was still willing to contribute to EACopy or not.

**Comment by ben-may** [comment]
@sergeevabc we have a fix incoming for this, sorry for the long delay!

**Comment by ben-may** [comment]
This is fixed in the latest [v1.14](https://github.com/electronicarts/EACopy/releases/tag/v1.14) release.
